pr-best-practices.md

```diff
@@ -11,4 +11,17 @@ Keep your pull requests as small as possible. We should be looking for ways to s
 - *"Are there sub-features of this Jira ticket that can be deployed in isolation and have benefit to the product before the rest of my work is done?"*
 - *"Perhaps I should merge lots of small pull requests into a separate branch for my feature and have those reviewed individually, and then merge that into edge once it's tested."*
 
-Smaller PRs are easier to review and easier to keep up to date with master/edge branches. There's no hard rule for this, obviously changing the same thing in a hundred files is a simple thing to review, and sometimes changes just won't be simple, but we should try and make them as easy to review as possible.
\ No newline at end of file
+Smaller PRs are easier to review and easier to keep up to date with master/edge branches. There's no hard rule for this, obviously changing the same thing in a hundred files is a simple thing to review, and sometimes changes just won't be simple, but we should try and make them as easy to review as possible.
+
+
+## Front end changes need SEO
```

Comment:
landing page sounds like a good way of describing them. By front end page i was referring to pages indexed by Google which typically are front end rather than back end pages but "landing page" also seems to fit that description nicely too, thanks ill update